### ISSUE TITLE
Provider of imported GKE cluster is shown as "Google GKE"

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#9253]
Internal Ref: SURE-6528

After upgrading from Rancher 2.6.9 -> 2.6.13, the provider of imported GKE cluster shown in GUI is modified from "Imported Google GKE" to "Google GKE". Rancher 2.7.4 shows same one.

Repro steps:

Import and create GKE cluster in Rancher 2.6.13.

Workaround:
Yes -- by running trailing command, we can get the GKE cluster is imported or not.

```
$ curl -k -u "<API Token>" -X GET "<hostname>/v3/clusters" | jq '.data[] | select(.name == "<cluster name>") | .gkeConfig.imported'
```

Actual behavior:
Imported GKE cluster provider is "Google GKE".

Expected behavior:
Imported GKE cluster provider is "Imported Google GKE".

**Comment by gaktive** [comment]
First step: investigate the behaviour with the upgrade and see what metadata is presented to us. This sounds like it's potentially backend but let's compare what we're getting between 2.6 & 2.7.

**Comment by cpinjani** [comment]
Validated as fixed on build: `v2.7-6bc6c907d60747d72894f00e97be0f9c3dfa3bbd-head` (tested on fresh install and upgraded from 2.6.13)

![image](https://github.com/rancher/dashboard/assets/73099870/1bb15d27-e1e2-4f15-ad8e-7d709e918c69)
